external/curl/Kconfig

```diff
@@ -0,0 +1,186 @@
+menu "CURL Options"
+
+config ENABLE_CURL
+	bool "Enable CURL"
+	default y
```

Comment:
CURL is not mandatory functionality to all. Or it is not our recommandation.
Let's set it as n.

Reply:
Okey